README.md

```diff
@@ -10,7 +10,11 @@ First, run this command to build the docs:
 docker run --rm --name slate -v $(pwd)/build:/srv/slate/build -v $(pwd)/source:/srv/slate/source slatedocs/slate
 ```
 
-Then, run the `deploy.sh` script to deploy them.
+Then, run this command to deploy them:
+
+```
+./deploy.sh --push-only
+```
```

Comment:
```suggestion
```
Without the `--push-only` option the deploy script tries to build the site first, which would require you to install a number of dependencies, which the docker command above made unnecessary.
```

Reply:
well, looks like i found a github bug. It gets confused when dealing with nested "```"s